examples/c11/z_scout.c

```diff
@@ -64,8 +64,8 @@ void fprinthello(FILE *stream, const z_hello_t *hello) {
     fprintf(stream, " }");
 }
 
-void callback(z_owned_hello_t hello, void *context) {
-    fprinthello(stdout, hello._value);
+void callback(z_owned_hello_t *hello, void *context) {
+    fprinthello(stdout, hello->_value);
```

Comment:
not necessary for this PR, but we need a loan function here (zenoh-c currently doesn't have the equivalent loan-type)

Reply:
In Pico we have it, so I am just using the z_loan now.